clubs/models.py

```diff
@@ -57,3 +64,33 @@ class Tag(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Membership(models.Model):
```

Comment:
Seems like the Membership meets the criteria of a “through model” in Django, basically a many to many relationship with extra metadata. Could clean up queries to make this explicit: https://docs.djangoproject.com/en/2.2/topics/db/models/#intermediary-manytomany

Reply:
I believe `Membership` already is a through model. See [Clubs members definition](https://github.com/pennlabs/clubs-backend/blob/8bc7a5a255f6753159e47c5b57c29bc66bb568d4/clubs/models.py#L34)

Reply:
`Membership` should already be a through model. For the various permissions checks, I directly use the Membership object instead of going through Clubs to avoid an additional lookup (club data doesn't matter in these cases, just the role field).

Reply:
Let me know if you're referring to some other query!

Reply:
Whoops. Didn’t see that! Sorry